### PR TITLE
Contract calls and integration fixes

### PR DESCRIPTION
**Pull request by yathishram**
### Description

- This PR updates contract calls and fixes apis needed for `core-js` integrations

### Modified

- `Events.sol` is a contract instead of library.
- `guardianProof` in `SafientMain.sol` is now an `external` function and not `internal`

### CODE REVIEW

File: src/contracts/SafientMain.ts [hunk 1]
```diff
@@ -96,11 +96,11 @@ export class SafientMain {
    * @param evidenceURI - IPFS URI pointing to the evidence submitted by the claim creator
    * @returns A transaction response
    */
-  createClaim = async (safeIdOnThreadDB: string, evidenceURI: string): Promise<TransactionResponse> => {
+  createClaim = async (safeIdOnThreadDB: string, evidenceURI: string): Promise<any> => {
```
Comment (koshikraj) [inline src/contracts/SafientMain.ts:99]: Can we not use `TransactionResponse` type here?

File: contracts/SafientMain.sol [hunk 2]
```diff
@@ -109,7 +116,7 @@ contract SafientMain is Safes, Claims, Guardians, IArbitrable {
         Types.RecoveryProof[] memory _guardianproof,
         string[] memory _secrets,
         string memory _safeId
-    ) internal returns (bool) {
```
Comment (koshikraj) [inline contracts/SafientMain.sol:112]: Does this have to be `external `?

File: contracts/SafientMain.sol [hunk 3]
```diff
@@ -11,12 +11,19 @@ import "./interfaces/IArbitrable.sol";
 contract SafientMain is Safes, Claims, Guardians, IArbitrable {
     IArbitrator public arbitrator;
     address public safientMainAdmin;
+    uint256 disputeId;
```
Comment (koshikraj) [inline contracts/SafientMain.sol:14]: You may delete this if not needed.

File: contracts/SafientMain.sol [hunk 4]
```diff
@@ -11,12 +11,19 @@ import "./interfaces/IArbitrable.sol";
...
 
     constructor(IArbitrator _arbitrator) {
         arbitrator = _arbitrator;
         safientMainAdmin = msg.sender;
     }
 
+    // event CreateClaim(
```
Comment (koshikraj) [inline contracts/SafientMain.sol:21]: Delete the commented event.

File: src/contracts/SafientMain.ts (hunk 1 above)
Comment (yathishram) [reply src/contracts/SafientMain.ts:99]: Oh, yeah sorry forgot that. Will change that

File: contracts/SafientMain.sol (hunk 2 above)
Comment (yathishram) [reply contracts/SafientMain.sol:112]: Yeah, else the function is not available for making contract calls

File: contracts/SafientMain.sol (hunk 3 above)
Comment (yathishram) [reply contracts/SafientMain.sol:14]: Done

**Comment by koshikraj** [comment]
LGTM

**Pull request merged**